temporian/implementation/numpy/data/display_utils.py

```diff
@@ -365,9 +389,16 @@ def display_text(evset: EventSet) -> str:
             ):
                 index_key_repr.append(f"{index_name}={index_value}")
             index_key_repr = " ".join(index_key_repr)
+            timestamps = index_data.timestamps
+            if is_datetime:
+                timestamps = np.array(
+                    convert_timestamps_to_datetimes(timestamps),
```

Comment:
You're right, actually there's no need to normalize these timestamps, they're part of the EventSet and should be float64 (otherwise, we've other greater issues).
Just calling `timestamps.astype('datetime64[s]')` is enough. It converts the whole array but that's not too expensive. Converting only the displayed events is not trivial and would require a new test (overkill, imo).